README.md

```diff
@@ -20,7 +20,6 @@
 
 <br>
 <br>
-
```

Comment:
This breaks markdown formatting.

Reply:
haven't touched. everything was merged from main branch. and accepted all incoming.